src/components/Ilkhom.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+
+export const Ilkhom = () => {
+    const [randomNumber, setRandomNumber] = useState(null)
+    const [userInput, setUserInput] = useState('')
+    const [result, setResult] = useState('')
+
+    const generateRandomNumber = () => {
+        const randomNum = Math.floor(Math.random() * 11);
+        setRandomNumber(randomNum);
+        console.log(randomNum);
```

Comment:
Remove this `console.log()`
![image](https://github.com/user-attachments/assets/333ee131-1a39-4dab-906a-66c44a6a0d73)